lib/tasks/grade.rake

```diff
@@ -53,18 +53,22 @@ namespace :grade do
         end
       end
     end
+    
+    if token.present? 
+      if is_valid_token?(submission_url, token) == false
+        puts "Access token is wrong. Please click on the assignment link again and run the rails grade ...  command shown there."
```

Comment:
@murugan-r What do you think about if `is_valid_token?(submission_url, token) == false`, we delete `grades.yml`) and then say

```ruby
puts "Your access token looked invalid, so we've reset it to be blank. Please re-run rails grade, and copy-paste your token carefully from the assignment page."
```

Reply:
@raghubetina Instead of deleting the grades.yml we can just set the personal_access_token to nil.